packages/web/src/common/store/pages/signon/sagas.ts

```diff
@@ -629,13 +639,6 @@ function* createGuestAccount(
     yield* call(fetchAccountAsync, { isSignUp: true })
 
     const userBank = yield* call(getOrCreateUSDCUserBank)
-    yield* put(
```

Comment:
why's this not needed anymore? is this to enable pay-with-anything?

Reply:
this was actually unnecessary because fetchAccountAsync handles this already. i had added this while testing out adding solana bank accounts to this payload but that wasn't necessary either